src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const bodyElement = document.body;
+
+const dataSalary = [...bodyElement.querySelectorAll('li[data-salary]')];
+const listPeople = [...bodyElement.querySelectorAll('li')];
+
+const salaries = dataSalary
+  .map((elem) => +elem.getAttribute('data-salary').replace(/[$,]+/g, ''))
+  .sort((a, b) => b - a);
+
+const sortedArray = [];
+
+function sortList(list) {
+  for (let i = 0; i < list.length; i++) {
+    const salary = salaries[i];
+
+    const person = list.filter(
+      (elem) =>
+        +elem.getAttribute('data-salary').replace(/[$,]+/g, '') === salary,
+    );
+
+    person.forEach((elem) => {
+      if (!sortedArray.includes(elem.textContent)) {
+        sortedArray.push(elem.textContent);
+      }
+    });
+  }
+}
+
+function getEmployees(list) {
+  for (let i = 0; i < list.length; i++) {
+    list[i].textContent = sortedArray[i] || '';
```

Comment:
The assignment 'list[i].textContent = sortedArray[i] || ''' might lead to unexpected results if 'sortedArray[i]' is an empty string or any falsy value. Ensure that this behavior is intended.